pixeltable/metadata/schema.py

```diff
@@ -252,7 +287,7 @@ class Table(Base):
 
 
 @dataclasses.dataclass
-class TableVersionMd:
+class VersionMd:
```

Comment:
Also change `TableSchemaVersionMd` to `SchemaVersionMd` for consistency. (TableSchemaVersionMd is a bit of a mouthful anyway)

Reply:
Done.